Bundle/CoreBundle/Resources/doc/SETUP.md

```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
+
+Register the following bundles in the `AppKernel`:
+
+`app/config/AppKernel.php`
+```php
+<?php
+use Symfony\Component\HttpKernel\Kernel;
+
+class AppKernel extends Kernel
+{
+    public function registerBundles()
+    {
+        $bundles = [
+            ...
+            //dependencies
+            new Troopers\AsseticInjectorBundle\TroopersAsseticInjectorBundle(),
+            new Troopers\AlertifyBundle\TroopersAlertifyBundle(),
+            new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
+            new FOS\UserBundle\FOSUserBundle(),
+            new FOS\JsRoutingBundle\FOSJsRoutingBundle(),
+            new JMS\SerializerBundle\JMSSerializerBundle(),
+            new Liip\ImagineBundle\LiipImagineBundle(),
+            new Knp\Bundle\MenuBundle\KnpMenuBundle(),
+            new Knp\DoctrineBehaviors\Bundle\DoctrineBehaviorsBundle(),
+            new Snc\RedisBundle\SncRedisBundle(),
+            new Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle(),
+            //Victoire bundles
+            new Victoire\Bundle\AnalyticsBundle\VictoireAnalyticsBundle(),
+            new Victoire\Bundle\BlogBundle\VictoireBlogBundle(),
+            new Victoire\Bundle\BusinessEntityBundle\VictoireBusinessEntityBundle(),
+            new Victoire\Bundle\BusinessPageBundle\VictoireBusinessPageBundle(),
+            new Victoire\Bundle\CoreBundle\VictoireCoreBundle(),
+            new Victoire\Bundle\CriteriaBundle\VictoireCriteriaBundle(),
+            new Victoire\Bundle\FilterBundle\VictoireFilterBundle(),
+            new Victoire\Bundle\FormBundle\VictoireFormBundle(),
+            new Victoire\Bundle\I18nBundle\VictoireI18nBundle(),
+            new Victoire\Bundle\MediaBundle\VictoireMediaBundle(),
+            new Victoire\Bundle\PageBundle\VictoirePageBundle(),
+            new Victoire\Bundle\QueryBundle\VictoireQueryBundle(),
+            new Victoire\Bundle\SeoBundle\VictoireSeoBundle(),
+            new Victoire\Bundle\SitemapBundle\VictoireSitemapBundle(),
+            new Victoire\Bundle\TemplateBundle\VictoireTemplateBundle(),
+            new Victoire\Bundle\TwigBundle\VictoireTwigBundle(),
+            new Victoire\Bundle\UserBundle\VictoireUserBundle(),
+            new Victoire\Bundle\ViewReferenceBundle\ViewReferenceBundle(),
+            new Victoire\Bundle\WidgetBundle\VictoireWidgetBundle(),
+            new Victoire\Bundle\WidgetMapBundle\VictoireWidgetMapBundle(),
+            ...
+        ];
+    }
+```
+
+### Start a redis server
+
+[https://redis.io/topics/quickstart](Quick start)
+
+just start it with docker:
+```sh
+docker run -d -p 6385:6379 --name myAwesomeRedis redis:latest
+```
+
+### Config
+
+#### Enable the serializer and translator
+
+`app/config/config.yml`
 ```yml
-twig.extension.intl:
-    class: Twig_Extensions_Extension_Intl
-    tags:
-        - { name: twig.extension }
+framework:
+   ...
+   translator:      { fallbacks: ["%locale%"] }
+   serializer: { enable_annotations: true }
```

Comment:
I think the serializer is only used with the SearchWidget. If so, this line belongs to the SearchWidget doc